server/service/src/sync/test/test_data/invoice.rs

```diff
@@ -84,7 +84,14 @@ const TRANSACT_1: (&'static str, &'static str) = (
       "user4": "",
       "user_ID": "",
       "wardID": "",
-      "waybill_number": ""
+      "waybill_number": "",
+      "om_allocated_datetime": "",
+      "om_picked_datetime": null,
+      "om_shipped_datetime": "",
+      "om_delivered_datetime": "",
+      "om_verified_datetime": "",
+      "om_created_datetime": "",
+      "om_transport_reference": ""
```

Comment:
Arguably should break out into 3 test JSON:
- Where the `datetime` string is `""`
- `datetime` string is `null`
- the properly is outright missing entirely, i.e. no key.

Proooobably don't need to do that exhaustively for _every_ field though...

Reply:
Think its ok since you mixed the null case in.